input/projects/why-join-wip.md

```diff
@@ -0,0 +1,56 @@
+# Why should my project join the .NET Foundation?
+
+The .NET Foundation offers administrative and technical services to open source projects to help maintainers govern and develop their projects.
+
+.NET Foundation services aim to increase project stability and maturity and don't replace developers on member projects. Day-to-day development remains in the hands of project maintainers and contributors.
```

Comment:
.NET Foundation services aim to increase project stability and maturity. It does not replace developers on member projects. Day-to-day development remains in the hands of project maintainers and contributors.

Reply:
Thanks